### PR TITLE
Mentor mentee relation

### PR DESCRIPTION
**Pull request by justinlim1**
Added a class to retrieve and update mentor-mentee relationship on Firebase. Button to send mentor request is functional now.

### CODE REVIEW

File: ReferralPlease/ReferralPlease/HomePageViewController.swift [hunk 1]
```diff
@@ -26,6 +28,8 @@ class HomePageViewController: UIViewController, UITableViewDataSource {
         db = Firestore.firestore()
         
         tableView.dataSource = self
+
+        MentorRequests.update("id")
```
Comment (narrator0) [inline ReferralPlease/ReferralPlease/HomePageViewController.swift]: @justinlim1 Is this PR finished and ready to merge? Not sure what this line means.

Comment (justinlim1) [reply ReferralPlease/ReferralPlease/HomePageViewController.swift]: yeah merge it. This line is just a place holder to test that class until the user id is properly passed to the vc

Comment (narrator0) [reply ReferralPlease/ReferralPlease/HomePageViewController.swift]: You can use `User.currentUser()` to get the current logged in user. I not sure how to test this without seeing something working.

**Review by narrator0 — APPROVED** [review]
LGTM!

**Pull request merged**